tools/dhtnode.cpp

```diff
@@ -29,7 +29,7 @@ extern "C" {
 using namespace dht;
 
 void print_usage() {
-    std::cout << "Usage: dhtnode [-p local_port] [-b bootstrap_host:port]" << std::endl << std::endl;
+    std::cout << "Usage: dhtnode [-h] [-v [logfile]] [-i] [-d] [-n network_id] [-p local_port[:port]] [-b [bootstrap_host]]" << std::endl << std::endl;
```

Comment:
- ":port" is not part of "-p"
- "-h" : it's definitely not part of normal usage, can't be used with other options, and I think shouldn't be included.